unstructured/partition/common.py

```diff
@@ -569,6 +569,14 @@ def document_to_element_list(
                 element.metadata.text_as_html = (
                     layout_element.text_as_html if hasattr(layout_element, "text_as_html") else None
                 )
+                try:
+                    if (isinstance(element, Title) and not element.metadata.category_depth) and any(
+                        el.type in ["Headline", "Subheadline"] for el in page.elements
+                    ):
+                        element.metadata.category_depth = 0
+                except AttributeError:
+                    # HTML instances
```

Comment:
Could we add a logging statement here probably at the info level? Worried that passing this exception may hide a side effect that may be difficult to debug

Reply:
done